lib/spells/effects/RemoveObstacle.h

```diff
@@ -28,7 +28,7 @@ class RemoveObstacle : public LocationEffect
 {
 public:
 	RemoveObstacle();
-	virtual ~RemoveObstacle();
+	virtual ~RemoveObstacle() override = default;
```

Comment:
same

Reply:
Destructors can be marked as override? Nice, but pointless since base class has virtual destructor anyway.

Reply:
removed